.github/workflows/cli.yml

```diff
@@ -13,6 +13,13 @@ on:
       - "cli/**"
       - ".github/workflows/cli.yml"
 
+# Fix for OOM.
+env:
+  CARGO_BUILD_JOBS: 1
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: >
+    -C codegen-units=1
```

Comment:
From the [rustc book](https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units):

> Setting this to 1 may improve the performance of generated code, but may be slower to compile.

I see that you have also removed the "Increase swap file" step - are these related? How does this flag affect memory usage?

Reply:
well, CI passes so it seems it's affected positively. We agreed with @tuky191 that this is good for now as it works and he'll revise this when he has more time